src/man/sssd.conf.5.xml

```diff
@@ -423,6 +423,11 @@
                                     responder.
                                 </phrase>
                             </para>
+                            <para>
+                                Both a user name and a uid can be used but the
+                                user should be present in
+                                <filename>/etc/passwd</filename>.
```

Comment:
I think we shouldn't reference specific file location explicitly.
Imo, something like `but the user should be a local one, i.e. accessible via "files" service of "nsswitch.conf"` would be better.

Reply:
Btw, I guess some other man pages also could be updated (like `man sssd-ifp::allowed_uids`)

Reply:
Do you think that I should also update `sssd.conf.5.xml::allowed_uids`?

Reply:
That's what I meant, just not sure if it's worth the effort to use includes to avoid copy&paste.

Reply:
I don't think it's worth the effort as I only found that occurrence.